engine/src/flutter/lib/web_ui/flutter_js/src/entrypoint_loader.js

```diff
@@ -174,7 +174,17 @@ export class FlutterEntrypointLoader {
         importsPromise = Promise.resolve({});
       }
       const compiledDartApp = await compiledDartAppPromise;
-      const dartApp = await compiledDartApp.instantiate(await importsPromise);
+      const dartApp = await compiledDartApp.instantiate(await importsPromise, {
+        loadDynamicModule: async (wasmUri, mjsUri) => {
+          const wasmBytes = fetch(resolveUrlWithSegments(wasmUri));
+          let mjsRuntimeUri = resolveUrlWithSegments(entrypointBaseUrl, mjsUri);
```

Comment:
Is there some documentation about this that I can look at? I'm trying to understand why `mjsUri` is prefixed by `entrypointBaseUrl` but not `wasmUri`.

Reply:
Ah, okay, that makes more sense :)

Thanks for the link!